src/tests/Microsoft.PowerFx.Interpreter.Tests/InterpreterExpressionTestCases/DateOverride.txt

```diff
@@ -1,14 +0,0 @@
-﻿#OVERRIDE: Date.txt
-
-// dates before 1900 get converted to post-1900
->> Text(Date(80, 1, 2))
-"1/2/0080"
```

Comment:
@CarlosFigueiraMSFT  - are you saying keep this test case?

Reply:
yep - as a "specification" that the C# interpreter has a different behavior compared to Power Apps Canvas Apps (at this point)